backend/src/main/java/gov/cdc/usds/simplereport/db/model/Organization.java

```diff
@@ -1,132 +1,39 @@
 package gov.cdc.usds.simplereport.db.model;
 
-import java.util.Collection;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
-import java.util.stream.Collectors;
-
 import javax.persistence.Column;
 import javax.persistence.Entity;
-import javax.persistence.FetchType;
-import javax.persistence.JoinColumn;
-import javax.persistence.JoinTable;
-import javax.persistence.ManyToMany;
-import javax.persistence.ManyToOne;
-import javax.persistence.OneToOne;
 
 import org.hibernate.annotations.NaturalId;
+import org.springframework.boot.context.properties.ConstructorBinding;
 
 @Entity
 public class Organization extends EternalEntity {
 
 	@Column(nullable = false, unique = true)
-	private String facilityName;
+	private String organizationName;
 
 	@Column(name="organization_external_id", nullable=false, unique=true)
 	@NaturalId
 	private String externalId;
 
-	@Column(nullable=false)
-	private String cliaNumber;
-
-	@ManyToOne(optional = true)
-	@JoinColumn(name = "default_device_type")
-	private DeviceType defaultDeviceType;
-
-	@OneToOne(optional=false)
-	@JoinColumn(name = "ordering_provider_id", nullable = false)
-	private Provider orderingProvider;
-
-	@ManyToMany(fetch = FetchType.EAGER)
-	@JoinTable(
-		name = "facility_device_type",
-		joinColumns = @JoinColumn(name="organization_id"),
-		inverseJoinColumns = @JoinColumn(name="device_type_id")
-	)
-	private Set<DeviceType> configuredDevices = new HashSet<>();
-
 	protected Organization() { /* for hibernate */ }
 
-	public Organization(String facilityName, String externalId, String cliaNumber) {
+	@ConstructorBinding
+	public Organization(String orgName, String externalId) {
 		this();
-		this.facilityName = facilityName;
+		this.organizationName = orgName;
 		this.externalId = externalId;
-		this.cliaNumber = cliaNumber;
-	}
-
-	public Organization(String facilityName, String externalId, String cliaNumber, DeviceType defaultDeviceType, Provider orderingProvider) {
-		this(facilityName, externalId, cliaNumber);
-		this.defaultDeviceType = defaultDeviceType;
-		if (defaultDeviceType != null) {
-			this.configuredDevices.add(defaultDeviceType);
-		}
-		this.orderingProvider = orderingProvider;
 	}
 
-	public Organization(
-			String facilityName,
-			String externalId,
-			String cliaNumber,
-			DeviceType defaultDeviceType,
-			Provider orderingProvider,
-			Collection<DeviceType> configuredDevices) {
-		this(facilityName, externalId, cliaNumber, defaultDeviceType, orderingProvider);
-		this.configuredDevices.addAll(configuredDevices);
+	public String getOrganizationName() {
+		return organizationName;
 	}
 
-	public void setFacilityName(String facilityName) {
-		this.facilityName = facilityName;
-	}
-
-	public String getFacilityName() {
-		return facilityName;
+	public void setOrganizatioName(String newName) {
```

Comment:
nitpick `setOrganizatioName ` -> `setOrganizationName `

Reply:
missing the n at the end of Organization

Reply:
Thanks for the second comment, I could still barely see the typo after you pointed it out. &lt;sigh&gt;